### DISCUSSION TITLE
Where do you discuss about contribution?

### DISCUSSION THREAD

**Discussion by aregtech** [community/community#67193] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi community,
Where do you discuss or ask for help to contribute in a project? Do you have such talks?

**Comment by queenofcorgis** [comment, marked as answer]
Hi there!

Check out our new [Programming Help](https://github.com/orgs/community/discussions/categories/programming-help) 🧑‍💻 category, for discussing and getting feedback on Projects. 

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.

**Reply by aregtech** [reply]
OK. Thank you. I'll use this category.
